doc/contribute-docs.md

```diff
@@ -1,25 +1,78 @@
 # How to contribute documentation
+This guide provides all the information necessary to contribute to the Netplan documentation, especially if you're contributing for the first time.
 
 ## Reporting an issue
 
-If you find any issue in Netplan documentation please [file a bugreport](https://bugs.launchpad.net/netplan/+filebug?field.tags=documentation)
-about it in our bug tracker on Launchpad. Remember adding a `documentation` tag
+To report an issue in Netplan documentation, [file a bug](https://bugs.launchpad.net/netplan/+filebug?field.tags=documentation) about it in our bug tracker on Launchpad. Remember to add a `documentation` tag.
 to it.
 
 ## Modifying documentation online
 
-Each documentation page rendered on the web contains an **Edit this page** link
-in the top-right of every page. Clicking this button will lead you to the GitHub
+Each documentation page rendered on the web contains an **Edit this page** link in the top-right corner. Clicking this button leads you to the GitHub
 web editor where you can propose changes to the corresponding page.
 
 Please remember to first check the [latest version](https://netplan.readthedocs.io/en/latest/)
 of our documentation and make your proposal based on that revision.
 
-## Creating a pull request
+## Contributing on GitHub
 
-If you want to follow a Git development workflow, you can also checkout the
+To follow a Git development workflow, `checkout` the
 [Netplan repository](https://github.com/canonical/netplan) and contribute your
 changes as [pull requests](https://github.com/canonical/netplan/pulls), putting
 the `documentation` label for better visibility.
```

Comment:
Can non-admins add labels to PRs?

Reply:
Hi, @rkratky. I don't think so. It was part of the document before I made changes. I'll remove it right away. Thank you for spotting it.